runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceContent/OpenSourceContent.module.scss

```diff
@@ -0,0 +1,114 @@
+@import '#src-landing/scss/main.scss';
+
+$ICON_SIZE: 76px;
+$SEPARATOR_FIRST_DIM: 2px;
+$SEPARATOR_SECOND_DIM: 36px;
+
+.content {
+    width: 100%;
+    max-width: 1672px;
```

Comment:
why that size

Reply:
from SectionContent.module.scss, so I'm just gonna use it xd